src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/CoordinatesFormattingTest.java

```diff
@@ -0,0 +1,69 @@
+/* (c) 2018 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.test;
+
+import org.geoserver.catalog.FeatureTypeInfo;
+import org.geotools.feature.NameImpl;
+import org.junit.Test;
+import org.w3c.dom.Document;
+
+/** Testing for Coordinates formatting configurations on WFS GML 3.1 & 3.2 on complex features */
+public class CoordinatesFormattingTest extends StationsAppSchemaTestSupport {
```

Comment:
Nothing needs to be changed, just sharing some thoughts. 

Every time a test like this is added it will add an _avoidable_ test execution overtime, the data set will need to be setup and a GeoServer mocked instantiated started (this is expensive).  We need to find a way to organize WFS tests in a more eficient way.